.devcontainer/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
This seems against good practices in Docker.
To  use an "entrypoint" for the purpose of validating the Dockerfile, even if conditional.
I would do that with  a github action:
```
docker-check:
      name: docker
      runs-on: ubuntu-latest
      steps:
      - uses: actions/checkout@v2
        with:
          token: ${{ secrets.GH_PAT || github.token }}

      - name: Build Dockerfile
        run: docker build -t vega-lite .
        
      - name: Run yarn-build
        run: docker run -ti -v `pwd`:/app -w /app vega-lite yarn build
```

Reply:
Good point. I took a different approach and built the docker file as you suggested.